src/test/java/io/github/wimdeblauwe/hsbt/mvc/HxGetMapping.java

```diff
@@ -0,0 +1,22 @@
+package io.github.wimdeblauwe.hsbt.mvc;
+
+import org.springframework.core.annotation.AliasFor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target(ElementType.METHOD)
+@Retention(RetentionPolicy.RUNTIME)
+@HxRequest
+@RequestMapping(
```

Comment:
Would it work with using `@GetMapping` as well? Maybe a bit cleaner that way?

Reply:
I wasn't able to make it work. I bet there is a way, but in my short attempt I didn't see a way. I would be loads cleaner with `@GetMapping`. I'll post to stack overflow to see if anyone has an idea.

Reply:
Stack overflow question: https://stackoverflow.com/questions/72152632/spring-boot-how-to-create-a-composed-annotation-using-getmapping